HISTORY.rst

```diff
@@ -1,5 +1,11 @@
 ## History
 
+### 1.0.2
+
+* Added support for multilingual eval
+* Added support for linear probing
```

Comment:
there was already linear probing, just not the ability to run exps with python script